metron-interface/metron-rest/src/test/java/org/apache/metron/rest/controller/KafkaControllerIntegrationTest.java

```diff
@@ -61,6 +62,7 @@
   private static final int KAFKA_RETRY = 10;
   @Autowired
   private KafkaComponent kafkaWithZKComponent;
+  private ComponentRunner runner;
```

Comment:
I don't think so.  `setup()` uses kafkaWithZKComponent, which is `@Autowired`.  I don't think that can be static, so there's probably more refactoring involved.  I can look at it, if it you want.